codalab/rest/bundles.py

```diff
@@ -243,9 +247,10 @@ def _create_bundles():
     )
 
```

Comment:
For simplicity, let's set `parent_worksheet_required = not detached` and remove the `parent_worksheet_required` query parameter.

Reply:
Yeah, it makes sense based on the discussion! Just committed the update :)